hack/verify-staging-imports.sh

```diff
@@ -31,4 +31,9 @@ for dep in $(ls -1 ${KUBE_ROOT}/staging/src/k8s.io/); do
 	fi
 done
 
+if grep -rq '// import "k8s.io/kubernetes/' 'staging/'; then
```

Comment:
In fact, I misread your script. You only check for '// import "k8s.io/kubernetes". What I am proposing is to put `// import "k8s.io/apimachinery/pkg/genericapiserver/server"` into the staging/src/k8s.io/apimachinery packages. Will that break anything?

Btw, lgtm as a quick fix of course to fix go get.